### PR TITLE
style(item): remove empty space

### PR DESCRIPTION
**Pull request by s0urcelab**
remove left empty part for v-slide-group.

image

### CODE REVIEW

**Review by ThibaultNocchi — APPROVED** [review]
Tested and LGTM. You're just gonna need to fix the linting errors :)

**Comment by s0urcelab** [comment]
> Tested and LGTM. You're just gonna need to fix the linting errors :)

done!

**Review by ThibaultNocchi — APPROVED** [review]

File: client/pages/item/_itemId/index.vue [hunk 1]
```diff
@@ -392,3 +392,9 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style>
```
Comment (ferferga) [inline client/pages/item/_itemId/index.vue:396]: I think we want this scoped to this component only?

Comment (s0urcelab) [reply client/pages/item/_itemId/index.vue:396]: yep, but is there a way to inject custom style to Vuetify component?

Comment (ThibaultNocchi) [reply client/pages/item/_itemId/index.vue:396]: You don't have to, just use `<style scoped>` :)

**Comment by ThibaultNocchi** [comment]
@s0urcelab Have you looked at adding the scoped tag? :)

**Comment by ThibaultNocchi** [comment]
@jellyfin-bot rebase

**Comment by ThibaultNocchi** [comment]
Commits reused and scoped applied in #1160

**Pull request closed without merging**